### PR TITLE
Update parametric-functions.md

### PR DESCRIPTION
**Pull request by Warfields**
Fix link to pattern syntax

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Documentation entry for user-facing changes

- [X] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ]  Exclude: AST fuzzers
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by Warfields** [comment]
Fixes the broken section title (red highlight) and the links (green highlight)
image

File: docs/en/sql-reference/aggregate-functions/parametric-functions.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Events that occur at the same second may lay in the sequence in an undefined ord
 
 **Parameters**
 
-- `pattern` — Pattern string. See [Pattern syntax](#sequencematch).
+- `pattern` — Pattern string. See [Pattern syntax](#sequence-function-pattern-syntax).
```
Comment (yariks5s) [inline docs/en/sql-reference/aggregate-functions/parametric-functions.md:107]:
```suggestion
- `pattern` — Pattern string. See [Pattern syntax](#pattern-syntax).
```

File: docs/en/sql-reference/aggregate-functions/parametric-functions.md [hunk 2]
```diff
@@ -114,6 +114,7 @@ Events that occur at the same second may lay in the sequence in an undefined ord
 Type: `UInt8`.
 
 <a name="sequence-function-pattern-syntax"></a>
+
 **Pattern syntax**
```
Comment (yariks5s) [inline docs/en/sql-reference/aggregate-functions/parametric-functions.md:118]:
```suggestion
#### Pattern syntax
```

File: docs/en/sql-reference/aggregate-functions/parametric-functions.md [hunk 3]
```diff
@@ -196,7 +197,7 @@ sequenceCount(pattern)(timestamp, cond1, cond2, ...)
 
 **Parameters**
 
-- `pattern` — Pattern string. See [Pattern syntax](#sequencematch).
+- `pattern` — Pattern string. See [Pattern syntax](#sequence-function-pattern-syntax).
```
Comment (yariks5s) [inline docs/en/sql-reference/aggregate-functions/parametric-functions.md:200]:
```suggestion
- `pattern` — Pattern string. See [Pattern syntax](#pattern-syntax).
```

**Comment by yariks5s** [comment]
Previously, links that you made were not working. I remade them so they now are working as expected

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 76b938acaf236196dcfa502fd9b0e09190180f13 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/69055/76b938acaf236196dcfa502fd9b0e09190180f13/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/69055/76b938acaf236196dcfa502fd9b0e09190180f13/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

**Comment by Warfields** [comment]
Thanks @yariks5s for the help. Sorry about the delay, I just got back from vacation.

**Pull request merged**